.github/maintainers_guide.md

```diff
@@ -53,22 +53,25 @@ do so after you are done working in this project. To come back to development
 work for this project again in the future, `cd` into this project directory and
 run `source env_3.9.6/bin/activate` again.
 
-The last step is to install this project's dependencies; to do so, check out [how
-we configure GitHub Actions to install dependencies for this project for use in
-our continuous integration](https://github.com/slackapi/python-slack-sdk/blob/main/.github/workflows/ci-build.yml#L26-L30). You can also run `./scripts/run_validation.sh` to install the dependencies and run the unit tests in one command!
+The last step is to install this project's dependencies and run all unit tests; to do so, you can run
+```bash
+$ ./scripts/run_validation.sh 
+```
 
+Also check out [how
+we configure GitHub Actions to install dependencies for this project for use in
+our continuous integration](https://github.com/slackapi/python-slack-sdk/blob/main/.github/workflows/ci-build.yml#L26-L30).
 ## Tasks
 
 ### Testing (Unit Tests)
 
 When you make changes to this SDK, please write unit tests verifying if the changes work as you expected. You can easily run all the tests by running the command. The `validate` command runs Flake8 (static code analyzer), Black (code formatter), and unit tests in the `tests` directory for you.
 
 ```bash
-python setup.py validate # run all
```

Comment:
Maybe in this code block we show three different commands:

1. Run all the unit tests, and code formatter, and code analyzer: `./scripts/run_validation.sh`
2. Run all the unit tests (but no formatter nor code analyzer): `./scripts/run_unit_tests.sh`
3. Run a specific unit test: `./scripts/run_unit_tests.sh tests/web/test_web_client.py`

Reply:
Let me know if [these changes](https://github.com/slackapi/python-slack-sdk/pull/1225/commits/573e5021d58a5ee5454fba253d178e6fcefc8667#diff-64710ddf9131ee0621e674aba34707aa85cca5f735c2c9421a7b6a57724e264eR70-R83) reflect what you had envisioned